core/domain/story_domain.py

```diff
@@ -1106,12 +1106,13 @@ def serialize(self) -> str:
             str. JSON-encoded str encoding all of the information composing
             the object.
         """
-        # Here, to_dict() method returns a general dictionary representation of
-        # domain object which do not contain properties like created_on and
-        # last_updated but MyPy expecting story_dict a dictionary which
-        # contain all the properties of domain object. That's why we explicitly
-        # changing the type of story_dict here which causes MyPy to throw error,
-        # thus to silent the error we added an ignore here.
+        # Here we use MyPy ignore because to_dict() method returns a general
```

Comment:
I think you'll need to define a new type for the augmented dict (and possibly use a different varname).

Reply:
I think I already defined and assigned the new typeddict, that's why MyPy throws an error. And that's why ignore is needed here.